src/telegram-client/telegram-client.ts

```diff
@@ -21,11 +21,11 @@ export class TelegramClient implements ITelegramClient {
   /**
    * Sets the action to do on ANY incoming text.
    */
-  public setOnAnyText(action: ((msg: any, match: string[]) => string)): void {
+  public setOnAnyText(action: ((msg: any, match: string[]) => string[])): void {
     this.bot.on("message", (msg: any, match: string[]) => {
       const output = action(msg, match);
       if (output) {
-        this.sendMessage(msg.chat.id, output);
+        output.forEach(out => this.sendMessage(msg.chat.id, out));
```

Comment:
A consideration would be to send one single message in which all messages are concatenated instead of sending them all individually, to reduce stress on the API and prevent spammy impressions.

Reply:
Maybe. But concatenated messages would probably get very large very quickly. This provides a more natural separation between the origin and nature of every individual message.